src/internal/easee/thing.go

```diff
@@ -120,6 +125,7 @@ func (t *thingFactory) chargepointSpecification(adapter adapter.Adapter, thingSt
 		t.supportedStates(),
 		chargepoint.WithChargingModes(SupportedChargingModes()...),
```

Comment:
As we discussed - service options should be set according to device capabilities.